advocacy_docs/pg_extensions/ldap_sync/installing.mdx

```diff
@@ -20,6 +20,10 @@ Before you begin the installation process:
 
   - [Installing EDB Postgres Extended Server](/pge/latest/installing/)
 
+- Install [pg_cron](https://github.com/citusdata/pg_cron)
+
+- Install plpython3u utility to run ldap2pg
```

Comment:
People might wonder how to install this.  This is the plpython3 package.  Not sure how we want to document this.  Maybe need a separate doc for it?

      epas: edb-as{{postgres_version}}-server-plpython3
      postgresql: postgresql{{postgres_version}}-plpython3
      edbpge: edb-postgresextended{{postgres_version}}-plpython3

Reply:
Okay, I made a ticket to capture the need for clearer instructions around installing plpython3u. https://enterprisedb.atlassian.net/browse/DOCS-276